cmd/containerd/main_windows.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"context"
+	"net"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"google.golang.org/grpc"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/docker/containerd/execution"
+	"github.com/urfave/cli"
+)
+
+func appendPlatformFlags(flags []cli.Flag) []cli.Flag {
+	return flags
+}
+
+func processRuntime(ctx context.Context, runtime string, root string) (execution.Executor, error) {
+	// TODO
+	return nil, nil
```

Comment:
@stevvooe FYI, the listener part will be addressed by a PR from https://github.com/jhowardmsft/containerd/tree/compile2, currently commit https://github.com/jhowardmsft/containerd/commit/a25092f03bcc703563a10ab9219c124098c74bf3